### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by steffenachim**

### CODE REVIEW

File: server/src/main/java/com/amos/server/EventGrabDemo.java [hunk 1]
```diff
@@ -87,7 +89,7 @@ public boolean onKey(View v, int k, KeyEvent e) {
                 int action = e.getAction();
                 int keycode = e.getKeyCode();
                 int meta = e.getMetaState();
-                msg = String.format("Keycode: %d Action: %d Meta: %d", keycode, action, meta);
+                msg = String.format(Locale.GERMAN,"Keycode: %d Action: %d Meta: %d", keycode, action, meta);
```
Comment (ghost) [inline server/src/main/java/com/amos/server/EventGrabDemo.java]: wouldn't we rather use English?

File: server/src/main/java/com/amos/server/EventGrabDemo.java [hunk 2]
```diff
@@ -66,7 +68,7 @@ public boolean onTouch(View v, MotionEvent e) {
                     id = e.getPointerId(i);
                     x = e.getX(i);
                     y = e.getY(i);
-                    msg = String.format("Count %d Index %d", count, i);
+                    msg = "Test Count "+count+" Index "+i;
```
Comment (ghost) [inline server/src/main/java/com/amos/server/EventGrabDemo.java]: why did you change this? String.format seems more elegant to me.

Comment (ghost) [reply server/src/main/java/com/amos/server/EventGrabDemo.java]: probably to circumvent *Warning: Implicitly using the default locale is a common source of bugs*

Comment (steffenachim) [reply server/src/main/java/com/amos/server/EventGrabDemo.java]: it's only a temporary demo class and i wanted to remove quickly the linting errors. So i think it's fine for this case ;)

File: server/src/main/java/com/amos/server/EventGrabDemo.java (hunk 1 above)
Comment (steffenachim) [reply server/src/main/java/com/amos/server/EventGrabDemo.java]: Same as below, but besides it doesnt makes a difference here.

Comment (xiamaz) [reply server/src/main/java/com/amos/server/EventGrabDemo.java]: Please use one or the other. It would be better not to hardcode locales.

Keep to using the same locale as in the printed text. In this case english would fit the best.

File: server/src/main/java/com/amos/server/EventGrabDemo.java (hunk 2 above)
Comment (xiamaz) [reply server/src/main/java/com/amos/server/EventGrabDemo.java]: Use locale here too if you are using them in other places as well. Also use locale english,

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]
